### PR TITLE
Add an endpoint to initialize the GUI

### LINKED ISSUE #59: Examine the handling of root directory selection in the GUI on Windows

### PR DESCRIPTION
**Pull request by brianhelba**
This should improve how we handle default directories on Windows. It also provides a better separation of concerns (GUI initialization vs actual directory selection), which should make future development and testing easier.

Fixes #59.

### CODE REVIEW

File: imagedephi/gui.py [hunk 1]
```diff
@@ -63,12 +63,26 @@ def reset_shutdown_event() -> None:
...
 
 
-@app.get("/", response_class=HTMLResponse)
+@app.get("/", response_class=RedirectResponse)
+def home(request: Request) -> str:
+    # On Windows, there may be multiple roots, so pick the one that's an ancestor of the CWD
+    # On Linux, this should typically resolve to "/"
+    root_directory = Path.cwd().root
+
+    # TODO: FastAPI has a bug where a URL object can't be directly returned here
```
Comment (brianhelba) [inline imagedephi/gui.py:72]: @brianhelba TODO: report this.

**Comment by brianhelba** [comment]
@marySalvi Beyond the two issues I've self-identified, can you review this and let me know if it's a reasonable approach?

**Review by marySalvi — APPROVED** [review]
LGTM

**Comment by naglepuff** [comment]
@marySalvi @brianhelba what is the status of this PR? Can it be closed?

**Comment by manthey** [comment]
This doesn't seem necessary with the current behavior on Windows.  We can reopen it if we decide it is needed.

**Pull request closed without merging**